src/Eppie.App/Eppie.App.ViewModels/LocalAIAgentSettingsPageViewModel.cs

```diff
@@ -527,8 +539,20 @@ private async Task UpdateAIAgentsListAsync()
                 AIAgentsList.SetItems(filteredAgents);
             }
 
+            var noneAgent = new LocalAIAgent() { Name = GetLocalizedString("NoneText") };
+            AIAgentsList.Insert(0, noneAgent);
+
             PreprocessorAIAgent = AIAgentsList.FirstOrDefault(agent => agent.Id == AgentSettingsModel.CurrentAgent.PreProcessorAgentId);
+            if (PreprocessorAIAgent == null)
+            {
+                PreprocessorAIAgent = noneAgent;
+            }
+
             PostprocessorAIAgent = AIAgentsList.FirstOrDefault(agent => agent.Id == AgentSettingsModel.CurrentAgent.PostProcessorAgentId);
+            if (PostprocessorAIAgent == null)
```

Comment:
may be `PostprocessorAIAgent` rename to `PostProcessorAIAgent` like everywhere else